content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -33,79 +33,88 @@ further_reading:
       tag: "GitHub"
       text: "Examples of Custom Instrumentation"
 ---
-## Compatibility Requirements
+## Compatibility requirements
 
 The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and above. For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
 
-## Getting started
+## Installation and getting started
 
 ### Follow the in-app documentation (Recommended)
 
 Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
 
 - Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
 - Dynamically set `service`, `env`, and `version` tags.
-- Enable ingesting 100% of traces and Trace ID injection into logs during setup.
+- Enable the Continuous Profiler, ingesting 100% of traces, and Trace ID injection into logs during setup.
 
-Otherwise, to begin tracing applications written in any language, first [install and configure the Datadog Agent][3]. The .NET Tracer runs in-process to instrument your applications and sends traces from your application to the Agent.
+### Automatic instrumentation
 
-**Note**: The .NET Tracer supports all .NET-based languages (C#, F#, Visual Basic, etc).
+<div class="alert alert-warning"> 
+  <strong>Note:</strong> Whether you're using custom or automatic instrumentation, it is important to keep the MSI installer and NuGet package versions in sync.
```

Comment:
This should read "When using both custom and automatic instrumentation, ..."

Reply:
Changed to avoid this phrasing altogether.